code/script.js

```diff
@@ -32,16 +37,177 @@ const showMessage = (message, sender) => {
 
```

Comment:
When there are many text-boxes, the buttons overlaps the text a bit. Add a function that makes the chat scroll to the last message, for example; chat.scrollTop = chat.scrollHeight.